packages/tools/kadena-cli/src/keys/commands/keysDelete.ts

```diff
@@ -0,0 +1,101 @@
+import type { Command } from 'commander';
+import { Option } from 'commander';
+import { z } from 'zod';
+
+import { PLAIN_KEY_DIR } from '../../constants/config.js';
+import { services } from '../../services/index.js';
+import type { CommandResult } from '../../utils/command.util.js';
+import { assertCommandError } from '../../utils/command.util.js';
+import { createCommand } from '../../utils/createCommand.js';
+import { createOption } from '../../utils/createOption.js';
+import { log } from '../../utils/logger.js';
+import { select } from '../../utils/prompts.js';
+import { keysOptions } from '../keysOptions.js';
+
+export const deleteKey = async (key: string): Promise<CommandResult<{}>> => {
+  try {
+    await services.filesystem.deleteFile(`${PLAIN_KEY_DIR}/${key}`);
+    return { success: true, data: {} };
+  } catch (error) {
+    return { success: false, errors: ['Failed to delete key'] };
+  }
+};
+
+export const deleteAllKeys = async (): Promise<CommandResult<{}>> => {
+  try {
+    await services.filesystem.deleteDirectory(PLAIN_KEY_DIR);
```

Comment:
some how i feel it's okay to keep the directory and delete only files but i'm also not sure which one we wanna follow here. if we choose to delete directory then it makes sense to do it for all sub commands delete.

Reply:
it could be done, but it will always check for the keys folder existence anyways, so it will always be created, so i see no value in keeping the empty dir?